### PR TITLE
Move about Abakus from faq to new page

### PR DESCRIPTION
**Pull request by Bestem0r**
By request from HS, the "Om Abakus" section in the FAQ page should be moved to a new dedicated "Om Abakus" page. This PR introduces such a page, in addition to improving the content's readability and visibility of Abakus' many groups.

Let me know if you have any suggestions or feedback! 😄

**Before:**
[Screencast from 2024-07-20 19-22-05.webm](https://github.com/user-attachments/assets/694d3489-57d4-4b7b-9ed1-ac023046bcf7)
**After:**
[Screencast from 2024-07-20 19-22-33.webm](https://github.com/user-attachments/assets/5608eae3-c93d-4a6b-972e-ae578eab12e6)

Resolves [ABA-968](https://linear.app/abakus-webkom/issue/ABA-968/add-about-abakus-page)

### CODE REVIEW

**Review by norbye — APPROVED** [review]
Personally a fan of using more rem than em and px where possible, but I'll leave that up to you.

Reviewed rather quickly - so no knitpicking and if it works it works™️

**Comment by norbye** [comment]
Oh and make sure to test how the navbar acts when the screen shrinks now that the content is wider

File: pages/om-abakus.tsx [hunk 1]
```diff
@@ -0,0 +1,251 @@
...
+            Gruppeansvarlig Inter (ansvarlig for Bedkom og Fagkom): Mona Helness
+          </li>
+          <li>
+            Gruppeansvarlig Media (ansvarlig for PR, readme og Webkom): Elena
+            Willmann
+          </li>
+          <li>
+            Gruppeansvarlig Sosial (ansvarlig for Arrkom, Koskom og LaBamba):
+            Maria Hurtado Beisvåg
+          </li>
```
Comment (falbru) [inline pages/om-abakus.tsx:138]: This should be updated to:
Leder; Kristoffer Krogh Wetterhus
Nestleder: Tiril Sjøberg
Økans: Aurora Johansen
Øvre styremedlem: Emrik SJølie Moe
Øvre styremedlem: Hanna Jacobsen
Øvre styremedlem: Ingeborg Lundamo Lien

There are no "Gruppeansvarlig" anymore. They have been replaced by Øvre styremedlem

File: pages/om-abakus.tsx [hunk 2]
```diff
@@ -0,0 +1,251 @@
...
+      <br />
+
+      <h3 className={styles.subTitle}>Komiteer</h3>
+      <p>
+        En komité er en gjeng som gjør forskjellig type arbeid for å drifte
+        linjeforeningen og arrangere faglige og sosiale arrangementer for
+        medlemmene. Hver komite tar gjerne opp 7-8 medlemmer hvert år, og
+        komitemedlemskap er en god sosial plattform, også for å bli kjent på
+        tvers av trinn. Det lar deg i tillegg gjøre meningsfullt arbeid for
+        andre, som bedrer studiehverdagen til Abakus&apos; mange medlemmer
```
Comment (falbru) [inline pages/om-abakus.tsx:155]: Perfectly written! The paragraph really brings forth the fun in volunteering for Abakus!

**Review by falbru — APPROVED** [review]
This is AWESOME! ♥♥♥

File: pages/om-abakus.tsx [hunk 3]
```diff
@@ -0,0 +1,251 @@
...
+    title: "PR",
+    description:
+      "PR-komiteen til Abakus er ansvarlig for å presentere Abakus på best mulig måte, både innad og utad. Dette innebærer å designe og lage alt fra promoteringsfilmer til plakater. De har hovedansvaret for å utvikle og vedlikeholde Abakus’ grafiske profil, og ansvar for linjeforeningens sosiale medier.",
+    contribution:
+      "Mange vil nok først og fremst gjenkjenne hvem som er faddere ved å se at de har på seg en egen fadder-t-skjorte. Denne er det PR som har designet og fått laget. I tillegg til dette har de laget alle headere til arrangementer på abakus.no, grafiske kalendere, o.l.",
+  },
+  {
+    title: "Webkom",
+    description:
+      "Webkom er komiteen som har det tekniske ansvaret for Abakus. Det er Webkom som har laget abakus.no, abanav.abakus.no og mye mer som er helt kritisk for at Abakus skal gå rundt.",
```
Comment (falbru) [inline pages/om-abakus.tsx:52]: ababart.abakus.no instead of abanav

File: pages/om-abakus.tsx (hunk 1 above)
Comment (norbye) [reply pages/om-abakus.tsx:138]: Øvrige styremedlemmer**

File: pages/om-abakus.tsx [hunk 4]
```diff
@@ -0,0 +1,243 @@
...
+          kontakten med hovedsamarbeidspartneren til Abakus, fordele penger i
+          linjeforeningen, holde kontakt med fakultetet og instituttene og
+          representere Abakus utad ved forskjellige anledninger.
+        </p>
+        <h4 className={styles.subTitle}>Roller i hovedstyret:</h4>
+        <ul>
+          <li>Leder: Kristoffer Krogh Wetterhus</li>
+          <li>Nestleder: Tiril Sjøberg</li>
+          <li>Økans: Aurora Johansen</li>
+          <li>Øvrig styremedlem: Emrik SJølie Moe</li>
```
[the remark this answers is not in the document]
Comment (Bestem0r) [reply pages/om-abakus.tsx:128]: Screenshot from 2024-07-26 23-16-00 🤡

**Pull request merged**